ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/CreateViewBasedReportModal.tsx

```diff
@@ -84,18 +80,48 @@ function CreateViewBasedReportModal({
         >
             <Flex gap={{ default: 'gapMd' }}>
                 <FlexItem>
-                    Export a view-based CSV report from this view using the filters you&apos;ve
-                    applied. Once completed, this report will be available in the one time reports
-                    queue until it is purged according to your retention settings.
+                    Generate a CSV report based on this view and the filters you&apos;ve applied.
+                    Once completed, this report will be available in the view-based reports section
+                    until it is purged according to your retention settings.
```

Comment:
The text needed updating because I noticed we were still using the term "one time" here

Reply:
If the text needs a contraction, let’s follow IBM style guidelines:
* replace `&apos;` entity reference for unicode 0027 APOSTROPHE
* with unicode 2019 RIGHT SINGLE QUOTATION MARK

Available for copy-paste in let’s (pardon pun)